src/specs/models/samples-bodies/CreateSampleFile.v2.yaml

```diff
@@ -0,0 +1,23 @@
+title: Create sample file
+description: 'Data required to create a new sample file'
+type: object
+properties:
+  sampleFileId:
+    type: string
+  size:
+    type: number
+  metadata:
+    description: Metadata to append to the s3 object on upload
+    type: object
+    properties:
+      cellrangerVersion:
+        type: string
+        oneOf:
+          - pattern: v2
+          - pattern: v3
+ 
+required:
+  - sampleFileId
```

Comment:
what about uploadStatus? Or do we add it later when we call the patch endpoint?

Reply:
when a sample file is being created what we know is that the upload is about to begin so by default we set `uploading`, there's a different endpoint (`PATCH`) to update the status